includes/scheduler/class-actor.php

```diff
@@ -89,6 +92,21 @@ public static function blog_user_update( $value = null ) {
 		return $value;
 	}
 
+	/**
+	 * Schedule Activities.
+	 *
+	 * @param string   $new_status New post status.
+	 * @param string   $old_status Old post status.
+	 * @param \WP_Post $post       Post object.
+	 */
+	public static function schedule_post_activity( $new_status, $old_status, $post ) {
```

Comment:
If I named this function, I would call it `maybe_schedule_profile_update`. You can take it or leave it :)

I do like splitting it out from the rest of the post updates.